.travis.yml

```diff
@@ -2,6 +2,9 @@ language: python
 python:
   - "2.7"
   - "3.3"
+  - "3.4"
+  - "3.5"
+  - "pypy3"
```

Comment:
Yea, it looks like Travis CI only does major version for PyPy.

http://docs.travis-ci.com/user/languages/python/#PyPy-Support